cmd/lncli/walletrpc_active.go

```diff
@@ -1193,6 +1195,191 @@ func listAddresses(ctx *cli.Context) error {
 	return nil
 }
 
+var signMessageWithAddrCommand = cli.Command{
+	Name:  "signmessage",
```

Comment:
I think we already have a cli named `signmessage`, so this may be changed into `signmessagewithaddr`.

Reply:
its part of the `lncli wallet addresses signmessage` and `lncli wallet addresses verifymessage`. It was a recommendation by guggero, and I find it more elegant than go with `signmessagewithaddr`

Reply:
oh yeah forgot about the subcommand and that's def that's much cleaner!